fdbclient/DatabaseContext.h

```diff
@@ -345,7 +345,7 @@ class DatabaseContext : public ReferenceCounted<DatabaseContext>, public FastAll
 	std::vector<std::unique_ptr<SpecialKeyRangeReadImpl>> specialKeySpaceModules;
 	std::unique_ptr<SpecialKeySpace> specialKeySpace;
 	void registerSpecialKeySpaceModule(SpecialKeySpace::MODULE module, SpecialKeySpace::IMPLTYPE type,
-	                                   std::unique_ptr<SpecialKeyRangeReadImpl> impl);
+	                                   std::unique_ptr<SpecialKeyRangeReadImpl> &&impl);
```

Comment:
Is this intended to be included into this PR?

Reply:
It isn't related, I just included it here because it didn't seem worth creating a brand new PR for this minor change.